modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceSalesAgent.testcase

```diff
@@ -211,6 +211,84 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-11277. As a Sales Agent, I can create a Quote Request in the Open Order's Details Page by entering a personal message."
+	@priority = 5
+	test CanCreateQuoteRequestAsSalesAgent {
+		property custom.properties = "feature.flag.COMMERCE-11028=true";
```

Comment:
@baharturk this overrides the custom.properties set at definition level, and it is causing the test to fail on CI

Reply:
Thanks, I'll move it out of the test when I fix it.